dashboard/app/views/projects/index.html.haml

```diff
@@ -69,6 +69,10 @@
       $scope.projectsLoaded = true;
     }).catch($scope.genericError);
 
+    $scope.projectVisible = function(project) {
+      return (!project.hidden);
```

Comment:
Either a `showHidden=true` param or maybe we really do need namespaces that can be used as filters in the .all() query.  This feels like scope creep so I'm fine with client-side for now, until we better understand what the API filters look like.